main.js

```diff
@@ -60,3 +60,113 @@ var writeLeaderBoard = function updateLeaderBoard() {
+  element.innerHTML = ''
+}
+function replacePlayerStats() {
+  $playerStats.innerHTML = ''
+}
+
+function createStatHeader() {
+  var createTable1 = document.createElement('table')
+  var create$tr1 = document.createElement('tr')
+  var create$th1 = document.createElement('th')
+  var create$th2 = document.createElement('th')
+  createTable1.className = 'table'
+  createTable1.id = 'player-stats-points'
+  create$th1.textContent = 'Stat'
+  create$th2.textContent = 'Score'
+  create$tr1.appendChild(create$th1)
+  create$tr1.appendChild(create$th2)
+  createTable1.appendChild(create$tr1)
+  return createTable1
+}
+
+function createMMRHeader() {
+  var createTable2 = document.createElement('table')
+  var create$tr2 = document.createElement('tr')
+  var create$th3 = document.createElement('th')
+  var create$th4 = document.createElement('th')
+  createTable2.className = 'table'
+  createTable2.id = 'player-stats-rank'
+  create$th3.textContent = 'GameMode'
+  create$th4.textContent = 'MMR'
+  create$tr2.appendChild(create$th3)
+  create$tr2.appendChild(create$th4)
+  createTable2.appendChild(create$tr2)
+  return createTable2
+}
+
+function createStatScore(statistics, i) {
+  var create$tr1 = document.createElement('tr')
+  var create$td1 = document.createElement('td')
+  var create$td2 = document.createElement('td')
+  var statArray = [
+    'Wins',
+    'Goals',
+    'MVPs',
+    'Saves',
+    'Shots',
+    'Assists',
+    statistics.stats.wins,
+    statistics.stats.goals,
+    statistics.stats.mvps,
+    statistics.stats.saves,
+    statistics.stats.shots,
+    statistics.stats.assists
+  ]
+  create$td1.textContent = statArray[i]
+  create$td2.textContent = statArray[(i + 6)]
+  create$tr1.appendChild(create$td1)
+  create$tr1.appendChild(create$td2)
+  return create$tr1
+}
+
+function createStatMMR(statistics, i) {
+  var create$tr1 = document.createElement('tr')
+  var create$td1 = document.createElement('td')
+  var create$td2 = document.createElement('td')
+  var statArray = [
+    'Solo Duel',
+    'Doubles',
+    'Solo Standard',
+    'Standard',
+    statistics.rankedSeasons[5][10].rankPoints,
+    statistics.rankedSeasons[5][11].rankPoints,
+    statistics.rankedSeasons[5][12].rankPoints,
+    statistics.rankedSeasons[5][13].rankPoints
+  ]
+  create$td1.textContent = statArray[i]
+  create$td2.textContent = statArray[(i + 4)]
+  create$tr1.appendChild(create$td1)
+  create$tr1.appendChild(create$td2)
+  return create$tr1
+}
+
+$search.addEventListener('keydown', function (event) {
+  if (event.keyCode === 13) {
+    for (var i = 0; i < data.length; i++) {
+      if ($search.value === data[i].displayName) {
+        var $tableStats = document.querySelector('#player-stats')
+        removeElement($leaderboard)
+        replacePlayerStats()
+        $tableStats.appendChild(createStatHeader())
+        $tableStats.appendChild(createMMRHeader())
+        var $tableStatsPoints = document.querySelector('#player-stats-points')
```

Comment:
Instead of querying the DOM for these elements, you should simply reference the return values of `createStatHeader` and `createMMRHeader` no? Aren't those the elements you're looking for?